nixos/modules/services/mail/dovecot.nix

```diff
@@ -118,9 +110,11 @@ let
 
     ''
       plugin {
+        ${concatStringsSep "\n" (mapAttrsToList (to: from: "sieve_${to} = ${stateDir}/sieve/${to}") cfg.sieve.scripts)}
         sieve_plugins = ${concatStringsSep " " cfg.sieve.plugins}
         sieve_extensions = ${concatStringsSep " " (map (el: "+${el}") cfg.sieve.extensions)}
         sieve_global_extensions = ${concatStringsSep " " (map (el: "+${el}") cfg.sieve.globalExtensions)}
```

Comment:
What do you think about making these and other plugin settings also use extraPluginSettings to reduce the amount of plaintext config?

Reply:
This would remove many options, right? And make the migration significant for other users?

Reply:
No, I'm talking about just setting `extraPluginSettings.sieve_plugins = "..."` in this module instead of putting it directly into the final config string. It shouldn't have significant (if any) impact on migration.

Reply:
Hmm, why not yes, but in that case, I guess we ought to call it `pluginSettings`.

Reply:
Kinda done.